packages/jellyfish-transaction/src/script/dftx/dftx.ts

```diff
@@ -250,6 +252,8 @@ export class CDfTx extends ComposableBuffer<DfTx<any>> {
         return compose<TakeLoan>(CTakeLoan.OP_NAME, d => new CTakeLoan(d))
       case CPaybackLoan.OP_CODE:
         return compose<PaybackLoan>(CPaybackLoan.OP_NAME, d => new CPaybackLoan(d))
+      case CCloseVault.OP_CODE:
```

Comment:
move to vault section

Reply:
This is the result of merge conflict. I am going to fix this in another PR.

Reply:
Fixed in updateVault DFTX PR